.github/workflows/pull_request_automation.yml

```diff
@@ -27,29 +27,35 @@ jobs:
       run: |
         flake8 . --config=.flake8 --show-source
 
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 4
+      matrix:
+        python-version: ['3.6', '3.7', '3.8', '3.9']
+
```

Comment:
Instead of defining 2 different jobs, we can combine them into a single one. Because otherwise we are having to duplicate the `checkout` and `install dependencies` step.

See the [`test.yml`](https://github.com/Eshaan7/click-creds/blob/main/.github/workflows/test.yml) in click-creds. It's a single job that checks linting as well as runs tests.